src/Sarif.FunctionalTests/SpecExamples/Comprehensive.sarif

```diff
@@ -28,20 +28,18 @@
         "startTime": "2016-07-16T14:18:25Z",
         "endTime": "2016-07-16T14:19:01Z",
         "machine": "BLD01",
-        "account": "builder",
+        "account": "build",
```

Comment:
Done.

---
In reply to: [72279460](https://github.com/Microsoft/sarif-sdk/pull/464#discussion_r72279460) [](ancestors = 72279460)